docs/tutorial/helloworld.py

```diff
@@ -0,0 +1,29 @@
+import time
+import http.server
+import os
+
+
+HOST_NAME = '0.0.0.0' # Host name of the http server
+# Gets the port number from $PORT0 environment variable
+PORT_NUMBER = int(os.environ['PORT0'])
+
+class MyHandler(http.server.BaseHTTPRequestHandler):
+    def do_GET(s):
+        """Respond to a GET request."""
+        s.send_response(200)
+        s.send_header("Content-type", "text/html")
+        s.end_headers()
+        s.wfile.write("<html><head><title>Time Server</title></head>")
+        s.wfile.write("<body><p>The current time is %s</p>" % time.asctime())
+        s.wfile.write("</body></html>")
+
+if __name__ == '__main__':
+    server_class = http.server.HTTPServer
+    httpd = server_class((HOST_NAME, PORT_NUMBER), MyHandler)
+    print (time.asctime(), "Server Starts - %s:%s" % (HOST_NAME, PORT_NUMBER))
```

Comment:
In Python 3, `print()` is a function so it's better not to have a space before the argument list. You can also use `str.format()` here:

```python
print(time.asctime(), "Server Starts - {}:{}".format(HOST_NAME, PORT_NUMBER))
```